### DISCUSSION TITLE
Display Template: Localized field?

### DISCUSSION THREAD

**Discussion by m1212e** [directus/directus#15715] [Archive]
Hi,
I've got blogposts which have localized texts & titles. In a one to many field inside another data model I want to make them selectable. Is it possible to display these localized titles inside the display template?

Inside the one to many selector settings: 
![grafik](https://user-images.githubusercontent.com/14091540/191837391-22f9e581-2e96-4c55-b5d4-fd91a3ba85d6.png)

What gets displayed when the title field is selected:
![grafik](https://user-images.githubusercontent.com/14091540/191836962-3e25ff89-040f-4e21-9e5d-e37c9396cad1.png)

It would help if I just could select index 0 from the localized titles array but a matching version to the current users language setting would be even better.

**Comment by azrikahar** [comment]
>  Is it possible to display these localized titles inside the display template?

It is possible 👍 You should change the O2M display template to `translations`, then make sure to configure said `translations` field to use the Translations Display. Here's a quick example to show what are the changes I meant:

https://user-images.githubusercontent.com/42867097/191898687-07379a66-957f-42d9-93dd-4b59bbdc7aaa.mp4

> but a matching version to the current users language setting would be even better.

As seen in the Translations display options, you can enable "Use Current User Language" option for this exact behavior!

**Reply by m1212e** [reply]
Ohh didn't really got how to do that until your video. Thanks a lot!
